src/libs/OptionsListUtils.ts

```diff
@@ -1769,6 +1846,8 @@ function filterAndOrderOptions(options: Options, searchInputValue: string, confi
         personalDetails: filteredPersonalDetails,
         userToInvite: filterResult.userToInvite,
         currentUserOption: filterResult.currentUserOption,
+        recentWorkspaceChats: filteredRecentWorkspaceChats,
+        selfDMChat: options.selfDMChat,
```

Comment:
We should also apply a filter to this option to prevent showing `No results found` while the list still includes this option.

![Screenshot 2024-12-14 at 12 20 49](https://github.com/user-attachments/assets/dc07df81-b960-4105-b204-a2ef0365d10f)